fixtures/import-order/.gitignore

```diff
@@ -7,3 +7,5 @@ dist/
 report/
 tsconfig.json
 # end managed by sku
+
+src/
```

Comment:
I'd occasionally end up with `src` files from this fixture hanging around because I interrupted the test before the cleanup step. This just ensures that you can't accidentally commit these files, because I definitely did that a few times.